packages/react/src/components/select/custom/custom-select.tsx

```diff
@@ -107,7 +107,10 @@ export function CustomSelect({
         {!currentValue || <option hidden value={currentValue} />}
       </NativeSelect>
 
-      <output className={classy(c('select-output'))}>{selectedOption}</output>
+      <output className={classy(c('select-output'))}>
+        {/* no-break space guarantees element height */}
+        {selectedOption}&nbsp;
```

Comment:
Maybe this is more noticable?

```suggestion
        {selectedOption}&nbsp;{/* no-break space guarantees element height */}
```

Reply:
Definitely, we can also put it in a new line for an even clearer effect

```suggestion
        {selectedOption}
        &nbsp; {/* no-break space guarantees element height */}
```